server/checks/Approval.js

```diff
@@ -139,9 +138,12 @@ export default class Approval extends Check {
             return
           }
           // get approvals for pr
-          const opener = pull_request.user.login
+          const commits = await github.fetchPullRequestCommits(user, repo, number, token)
+          const lastCommitter = commits.length === 0 ?
+            null :
+            commits[commits.length - 1].committer.login
           const comments = await github.getComments(user, repo, number, formatDate(dbPR.last_push), token)
-          const countConfig = Object.assign({}, config.approvals, {ignore: [opener]})
+          const countConfig = Object.assign({}, config.approvals, {ignore: lastCommitter ? [lastCommitter] : []})
```

Comment:
@whiskeysierra I think only ignoring the PR opener is not good enough from the compliance POV (as the approver might sneak in stuff by committing right before he approves, i.e. no four eyes).